services/github/github-tag.tester.js

```diff
@@ -28,6 +28,10 @@ t.create('Tag (repo not found)')
   .get('/v/tag/badges/helmets.json')
   .expectBadge({ label: 'tag', message: 'repo not found' })
 
+t.create('Tag (filter by prefix')
+  .get('/v/tag/expressjs/express.json?prefix=4.16')
+  .expectBadge({ label: 'tag', message: 'v4.16.4' })
```

Comment:
Same comment about this test as the release tester

Reply:
Same as above. I removed this and added a unit test